npm_tasks/build-core.js

```diff
@@ -0,0 +1,21 @@
+var fs = require('fs');
+var path = require('path');
+var execSync = require('child_process').execSync;
+
+var ROOT_DIR = path.resolve(__dirname, '..');
+var WEBMAKER_CORE_DIR = fs.realpathSync(path.resolve(
```

Comment:
This works really well, but does it not work without the  fs.realpathSync? (npm should be using the unicode path underwater, which is why node tools can create/delete dirs longer than 256 characters... I would think npm link deals with that, but then npm explore does really weird things too, so maybe not)